rest-api/src/main/java/uk/gov/gchq/gaffer/rest/application/ApplicationConfig.java

```diff
@@ -41,6 +42,7 @@ public ApplicationConfig() {
         addServices();
         setupBeanConfig();
         registerClasses(resources);
+        register(new FactoriesBinder());
```

Comment:
By having to register this in the ApplicationConfig anyone wanting to register a different User or Graph Factory, they will need to override this application config class, remove FactoriesBinder and add their own FactoriesBinder class. It seems like a lot more code than what they currently have to do. Currently users can just set the property class name for the factories.